atlp-rwanda/script-heroes-frontend
# #174797475 Comment on travel requests

**What does this PR do?**

Allows a user or  manager to post and  display comments relating to a request made

**Description of Task to be completed?**

   - Add comment UI component.
   - Add commenting functionality on requests
   - Test this feature

**How should this be manually tested?**

   - clone this repo `git clone https://github.com/atlp-rwanda/script-heroes-frontend`
   - checkout the branch` git checkout ft-comment-on-travel-requests-#174797475`
   - install dependencies `yarn install`
   - start the app` yarn run dev`
   - got to `/comments` to display all comments
   - check responsiveness on different screen sizes

**Any background context you want to provide?**

> N/A

**What are the relevant pivotal tracker stories?**

[#174797475](https://www.pivotaltracker.com/story/show/174797475)

**Screenshots (if appropriate)**

> N/A
